examples/p5js/Handpose/Handpose_Image/sketch.js

```diff
@@ -0,0 +1,52 @@
+let handpose;
+let predictions = [];
+let img;
+
+function setup() {
+  // Create a canvas that's at least the size of the image.
+  createCanvas(400, 350);
+
+  // create an image using the p5 dom library
+  // call modelReady() when it is loaded
+  img = createImg("data/hand.jpg", imageReady);
+
+  img.hide(); // hide the image in the browser
+  frameRate(1); // set the frameRate to 1 since we don't need it to be running quickly in this case
+}
+
+// when the image is ready, then load up poseNet
+function imageReady() {
+  handpose = ml5.handpose(img, modelReady);
```

Comment:
I might leave out the `img` in the model instantiation. Seems like it might be clearer to pass the `img` to `pedict()` so we can show how one might call predict on different images in a program?

So to extend the preload example:

```js

function preload(){
function preload(){
img = loadImage(....);
handpose = ml5.handpose();
}
}

function setup(){

  handpose.on("predict", results => {
    predictions = results;
  });


  handpose.predict(img);

}

function draw(){
// ...
}


```